src/gluonts/model/seq2seq/_forking_estimator.py

```diff
@@ -85,6 +94,10 @@ class ForkingSeq2SeqEstimator(GluonEstimator):
         seq2seq decoder
     quantile_output
         quantile output
+    distr_output
+        distribution output
+    sampling
+        True for sample prediction and False for distribution prediction
```

Comment:
We probably don't need the `sampling` option (at least for now): it's sufficient to have `quantile_output` specified or `distr_output` specified (but not both).